glossary.yml

```diff
@@ -1967,6 +1967,12 @@
       Sehemu ya habari inayowakilisha njia mbadala, ndiyo/hapana,
       [true](#true)/[false](#false). Katika kuhesabu hali ya 0 au 1.
 
+  uk:
+    term: "Біт"
```

Comment:
@VeronikaShevc why capitalise? Please see https://github.com/carpentries/glosario/pull/672#discussion_r1633889199 earlier this morning